.vscode/launch.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Environment files such as vs code or Pycharm should not be pushed to remote. Update `.gitignore`.